### DISCUSSION TITLE
TOTP context menu greyed out. Cannot add a new TOTP to every existing entry, but some are working...

### DISCUSSION THREAD

**Discussion by dieterlohrstraeter** [keepassxreboot/keepassxc#11043] [Q&A]
I'm wondering, why I cannot add a TOTP to most entries in my KeePassXC database.
The context menu is greyed out for most entries.
Interesstingly, for some entries it works and I can add.

What are the conditions to be fulfilled to get it enabled.

Just as an example: I want to add a new TOTP to my account on oracle.com.
They recently forced to use 2FA for my already existing account. I cannot add the TOTP to this entry as shown in screenshot:
![grafik](https://github.com/keepassxreboot/keepassxc/assets/16361139/29984096-2fc2-41db-854f-5803c69bdf9e)
It might look fine in the screenshot, but it's definitely greyed out!

Interestingly, the contextmenu is active for some other entries, but I don't see a clear difference.
See below a working entry:
![grafik](https://github.com/keepassxreboot/keepassxc/assets/16361139/7ad27b8e-9a50-4784-a092-ec28ac7c9396)

**Comment by droidmonkey** [comment]
https://github.com/keepassxreboot/keepassxc/issues/10981

Lock and unlock your database to regain functionality until this can be resolved. Your menu is missing the "Setup TOTP" entry though.

**Reply by dieterlohrstraeter** [reply]
Just did lock and unlock (via fingerprint), but it doesn't help. I've tried to stop the backend process and start again as well, but no success either. Can I do anything myself to create the missing "Setup TOTP" entry?

**Reply by droidmonkey** [reply]
I am definitely not seeing this problem on my end, Setup TOTP is always visible option

**Reply by dieterlohrstraeter** [reply]
Very strange: Now it works again!

The only thing I did: in "Database settings", I've removed a couple of old Browser integrations and saved it.
After saving, the "Setup TOTP" entry appeared again and is available for all entries.

Something seems to prevent this entry under specific conditions...
I've read [#10981](https://github.com/keepassxreboot/keepassxc/issues/10981), but to be honest, I'm not 100% sure, if it covers this problem. For now, it seems I found a workaround.

**Reply by WilliDieEnte** [reply]
Can confirm tho, had the same bug, restarting KeePassXC fixed it, and it seems to be #10981 :3

**Reply by lkjfdsaofmc** [reply]
> Very strange: Now it works again!
> 
> The only thing I did: in "Database settings", I've removed a couple of old Browser integrations and saved it. After saving, the "Setup TOTP" entry appeared again and is available for all entries.
> 
> Something seems to prevent this entry under specific conditions... I've read [#10981](https://github.com/keepassxreboot/keepassxc/issues/10981), but to be honest, I'm not 100% sure, if it covers this problem. For now, it seems I found a workaround.

I was having this same issue and I confirmed that I only had browser integrations saved for current browsers that I actively used, but after trying everything else listed here when I had the same issue I tried removing all browser integrations and that somehow resolved the issue for me.  I've re-added most of the browsers and it's still working, so not sure what caused this.
